corehq/sql_db/routers.py

```diff
@@ -125,13 +117,12 @@ def db_for_read_write(model, write=True, hints=None):
 
     if app_label == SYNCLOGS_APP:
         return settings.SYNCLOGS_SQL_DB_ALIAS
-    elif app_label == ICDS_REPORTS_APP:
-        return connection_manager.get_django_db_alias(ICDS_UCR_CITUS_ENGINE_ID)
-    elif app_label == AAA_APP:
-        engine_id = AAA_DB_ENGINE_ID
-        if not write:
-            return connection_manager.get_load_balanced_read_db_alias(AAA_DB_ENGINE_ID)
```

Comment:
this load balancing is removed but it's not active on any environment anyway. I think the original code was just copied from the ICDS code at the time.